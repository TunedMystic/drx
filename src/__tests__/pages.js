import renderWithRedux from '../utils/render-with-redux';

import About from '../pages/about';
import Cart from '../pages/cart';
import Index from '../pages/index';

describe('Page <About />', () => {
    it('should render correctly', () => {
        const { queryByText, container } = renderWithRedux(<About />);
        expect(container).toMatchSnapshot();

        expect(queryByText('About page')).toBeInTheDocument();
    });
});

describe('Page <Cart />', () => {
    it('should render correctly', () => {
        const { container } = renderWithRedux(<Cart />);
        expect(container).toBeInTheDocument();
    });
});

describe('Page <Index />', () => {
    it('Should render <Index />', () => {
        const { container } = renderWithRedux(<Index />);
        expect(container).toBeInTheDocument();

    });
});
