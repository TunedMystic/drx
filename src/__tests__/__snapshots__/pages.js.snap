// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page <About /> should render correctly 1`] = `
<div>
  <div>
    <div
      class="jsx-2087558186"
    >
      <div
        class="jsx-1059402298"
      >
        <a
          class="jsx-1059402298"
          data-testid="link-home"
          href="/"
        >
          Home
        </a>
        <a
          class="jsx-1059402298"
          data-testid="link-about"
          href="/about"
        >
          About
        </a>
        <a
          class="jsx-1059402298"
          href="/resource/1"
        >
          Resource
        </a>
      </div>
      <a
        class="jsx-2087558186"
        data-testid="link-cart"
        href="/cart"
      >
        Cart 
      </a>
    </div>
    <p>
      About page
    </p>
  </div>
</div>
`;
